packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -42,6 +42,17 @@ export interface ActionProperties {
    */
   readonly region?: string;
 
+  /**
+   * The account the Action is supposed to live in.
+   * For Actions backed by resources,
+   * this is inferred from the Stack {@link resource} is part of.
+   * However, some Actions, like the CloudFormation ones,
+   * are not backed by any resource, and they still might want to be cross-account.
+   * In general, a concrete Action class should specify either {@link resource},
```

Comment:
Is this enforced/validated?

Reply:
No. I don't think we should validate it actually. Imagine if someone vends an Action class that violates this rule. What will happen is that a customer will get an error when they use it -- but there will be no way for them to fix this problem, as the issue is in code they don't own.